test/mainWrapper/TestMainWrapper.gmk

```diff
@@ -0,0 +1,49 @@
+#
+# Copyright (c) 1997, 2022, Oracle and/or its affiliates. All rights reserved.
+# DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+#
+# This code is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License version 2 only, as
+# published by the Free Software Foundation.  Oracle designates this
+# particular file as subject to the "Classpath" exception as provided
+# by Oracle in the LICENSE file that accompanied this code.
+#
+# This code is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+# version 2 for more details (a copy is included in the LICENSE file that
+# accompanied this code).
+#
+# You should have received a copy of the GNU General Public License version
+# 2 along with this work; if not, write to the Free Software Foundation,
+# Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+#
+# Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+# or visit www.oracle.com if you need additional information or have any
+# questions.
+#
+
+#----------------------------------------------------------------------
+
+JTREG_OPTS += \
+	-mainWrapper:com.sun.javatest.regtest.agent.TestMainWrapper
+
+$(BUILDTESTDIR)/TestMainWrapper.ok: $(JTREG_IMAGEDIR)/lib/javatest.jar \
+		$(JTREG_IMAGEDIR)/lib/jtreg.jar \
+		$(JTREG_IMAGEDIR)/bin/jtreg
+	JTREG_JAVA=$(JDKJAVA) $(JTREG_IMAGEDIR)/bin/jtreg $(JTREG_OPTS) \
+		-jdk:$(JDKHOME) \
+		-w:$(@:%.ok=%)/cmd.othervm/work \
+		-r:$(@:%.ok=%)/cmd.othervm/report \
+		-verbose:fail \
+		$(TESTDIR)/mainWrapper
+	JTREG_JAVA=$(JDKJAVA) $(JTREG_IMAGEDIR)/bin/jtreg $(JTREG_OPTS) \
+		-jdk:$(JDKHOME) \
+                -agentvm \
+		-w:$(@:%.ok=%)/cmd.agentvm/work \
+		-r:$(@:%.ok=%)/cmd.agentvm/report \
+		-verbose:fail \
+		$(TESTDIR)/mainWrapper
+	echo "test passed at `date`" > $@
+
+INITIAL_TESTS += $(BUILDTESTDIR)/TestMainWrapper.ok
```

Comment:
I don't think this test is important enough to be in the set of initial tests; it should just be in the set of general jtreg tests (`TESTS.jtreg`)

Reply:
fixed